src/client/views/play.html

```diff
@@ -157,6 +157,8 @@
 			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
 		<script defer src="/scripts/game/chess/insufficientmaterial.js" onerror="htmlscript.callback_LoadingError(event)"
 			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+        <script defer src="/scripts/game/chess/premove.js" onerror="htmlscript.callback_LoadingError(event)"
```

Comment:
This addition of the script into play.html is actually **totally automated** now!

<img width="949" alt="Screen Shot 2024-07-19 at 1 56 53 AM" src="https://github.com/user-attachments/assets/9b393132-3df7-4996-8aff-6e53c7998c9a">

I just realized your fork is 143 commits behind the main branch! There are a lot of new changes you haven't seen yet! There should be a "sync" button on your fork page on github! You may have to resolve some conflicts if the main branch's code conflicts with your own.